src/Illuminate/Queue/Worker.php

```diff
@@ -148,7 +148,7 @@ protected function daemonShouldRun()
      */
     protected function waitForChildProcess($processId, $timeout)
     {
-        declare(ticks=1) {
+        declare (ticks = 1) {
```

Comment:
Yes I know, I'm just mentioning that there's a difference now between php fixer and style CI.